app/dispatches/players/change_nickname.rb

```diff
@@ -0,0 +1,33 @@
+module Players
+  class ChangeNickname
```

Comment:
Should a class name be a _noun_ or a _verb_? I tend toward the former, so _I_ would use `NicknameChange` instead. Just a matter of taste, though. I'm nitpicking to sound important.